### PR TITLE
docs: roadmap

### PR DESCRIPTION
**Pull request by sravankorumilli**

### CODE REVIEW

File: docs/docs/roadmap.md [hunk 1]
```diff
@@ -1,4 +1,50 @@
...
+### Authentication & Authorization in Optimus
+Optimus is multi-tenant, there is a need for proper auth through which users can be restricted to not interfere with the jobs that they don't own.
+There are some features like replay which has even a bigger impact & operations on these need to be highly restricted.
+
+### Secret Management in Optimus
+As optimus meant to deal with various warehouse systems & with the plugin support provides the capability to interact with other third party systems.
+It brings a need for proper secret management to store & use securely for all users onboard.
+
+### Test Your Jobs
+Giving a provision fo users to test the jobs before deploying helps users with faster feedbacks.
```
Comment (kushsharma) [inline docs/docs/roadmap.md:18]: `fo` typo

File: docs/docs/roadmap.md [hunk 2]
```diff
@@ -1,4 +1,50 @@
...
 id: roadmap
 title: Roadmap
----
\ No newline at end of file
+slug: /roadmap
+---
+
+This is a live document which gives an idea for all the users of Optimus on what are we upto. 
+
+### Authentication & Authorization in Optimus
```
Comment (kushsharma) [inline docs/docs/roadmap.md:9]: We decided to not support this in optimus I think?

**Comment by sravankorumilli** [comment]
the test failures are not related to this change @kushsharma are you aware of this issue.

**Comment by kushsharma** [comment]
> the test failures are not related to this change @kushsharma or @arinda-arif are you aware of this issue.

yeah, ignore that, that's a very annoying bug that only fails in github and we fail to reproduce it locally. This was introduced in one of the replay PR. I'll fix this soon, just need enough motivation.

**Pull request merged**